internal/util/fileop_test.go

```diff
@@ -217,6 +263,16 @@ func TestFileOpValidateBase(test *testing.T) {
 			nil,
 			"cannot point just to the current directory",
 		},
+		{
+			NewFileOperation([]string{"copy", "*/..", "b"}),
+			nil,
+			"cannot point just to the current directory",
+		},
+		{
+			NewFileOperation([]string{"copy", "*/[a-z", "b"}),
```

Comment:
Add a test case where this invalid pattern is used as a dest (and is therefore okay).
Currently this will fail, but we want it to pass
(which makes it a nice test).